semgrep/semgrep/util.py

```diff
@@ -64,3 +68,8 @@ def set_flags(debug: bool, quiet: bool) -> None:
     if quiet:
         QUIET = True
         debug_print("QUIET is on")
+
+
+def partition(pred: Callable, iterable: Iterable) -> Tuple[List, List]:
```

Comment:
Function level comment would be helpful. Not immediately clear what this function does